feat(Recipes): New Recipe ent-mysql

Eeap opened the pull request:
Hello, maintainers ! 
This is recipe to use ent ORM for Fiber with MySQL. I hope this recipe is useful 😊

gaby commented on ent-mysql/README.md:
```diff
@@ -0,0 +1,33 @@
+### Example ent ORM for fiber with MySQL
```
Can you please add the recipe to `.github/dependabot.yml` file

gaby commented on ent-mysql/go.mod:
```diff
@@ -0,0 +1,37 @@
+module http
+
+go 1.20
```
this can probably be `1.19`

gaby commented on ent-mysql/go.mod:
```diff
@@ -0,0 +1,37 @@
+module http
```
why not `ent-mysql` ?

gaby reviewed (changes requested):
Run the whole recipe through this:

```
gofumpt -l -w .
```

gaby commented:
@Eeap  Thanks for the submissions, just a few small changes :-) May also want to list the new recipe in the main README.md file

Eeap commented:
> @Eeap Thanks for the submissions, just a few small changes :-) May also want to list the new recipe in the main README.md file

Okay, Thank you for review. I have changed it

efectn commented on ent-mysql/routes/routes.go:
```diff
@@ -0,0 +1,71 @@
+	id, _ := strconv.Atoi(c.Params("id"))
+	b, err := database.DBConn.Book.
+		Get(ctx, id)
+	if err != nil {
```
We should check not found error here and return 404

Eeap replied:
> We should check not found error here and return 404

Oh, I see. I just added the error handling ! :)

efectn commented:
Why don't you check Post quieries when creating or updating? It should be better than hardcoded valuea

Eeap commented:
> Why don't you check Post quieries when creating or updating? It should be better than hardcoded values

I changed hardcoded values into post queries data and added empty check(if empty, return 400)

gaby commented:
@ReneWerner87 this one is also done, although now it has a conflict 😂

gaby commented:
@Eeap can take a look at the conflict, it's related to your other PR getting merged :-)

Eeap commented:
@gaby I checked that. Thanks for letting me know that conflict 😊
